### PR TITLE
Add missing test assertion for phx.gen.auth test

### PR DESCRIPTION
**Pull request by jsonmaur**

### CODE REVIEW

**Review by joshchernoff — APPROVED** [review]
LGTM :+1:

File: priv/templates/phx.gen.auth/registration_live_test.exs [hunk 1]
```diff
@@ -60,11 +60,14 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
...
-        <%= schema.singular %>: %{"email" => <%= schema.singular %>.email, "password" => "valid_password"}
-      )
-      |> render_submit() =~ "has already been taken"
+      result =
+        lv
+        |> form("#registration_form",
+          <%= schema.singular %>: %{"email" => <%= schema.singular %>.email, "password" => "valid_password"}
+        )
+        |> render_submit()
+
```
Comment (josevalim) [inline priv/templates/phx.gen.auth/registration_live_test.exs:69]:
```suggestion

```

File: priv/templates/phx.gen.auth/registration_live_test.exs [hunk 2]
```diff
@@ -60,11 +60,13 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
...
-      |> form("#registration_form",
-        <%= schema.singular %>: %{"email" => <%= schema.singular %>.email, "password" => "valid_password"}
-      )
-      |> render_submit() =~ "has already been taken"
+      result =
+        lv
+        |> form("#registration_form",
+          <%= schema.singular %>: %{"email" => <%= schema.singular %>.email, "password" => "valid_password"}
+        )
+        |> render_submit()
```
Comment (josevalim) [inline priv/templates/phx.gen.auth/registration_live_test.exs:68]:
```suggestion
        |> render_submit()

```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**